src/aldente_client.cpp

```diff
@@ -101,13 +102,18 @@ void AldenteClient::start() {
 
     std::cerr << "Starting client..." << std::endl;
 
+    // Used for callbacks
+    Timer timer(GAME_TICK);
```

Comment:
Since the catch up is non blocking, this essentially defines the quantization of timed callbacks to be 30ms.